src/main/java/racingcar/Car.java

```diff
@@ -1,10 +1,12 @@
 package racingcar;
 
 public class Car {
+	private final String name;
 	private int position;
 	private final CarMoveStrategy[] carMoveStrategies;
 
-	public Car(CarMoveStrategy... carMoveStrategies) {
+	public Car(String name, CarMoveStrategy... carMoveStrategies) {
+		this.name = name;
 		position = 0;
 		this.carMoveStrategies = carMoveStrategies;
```

Comment:
```suggestion
	public Car (String name, CarMoveStrategy... carMoveStrategies) {
		this.name = name;
		position = 0;
		this.carMoveStrategies = carMoveStrategies;
```
단축키를 이용해서 컨벤션을 지키면 좋겠습니다.